pysages/collective_variables/core.py

```diff
@@ -86,7 +86,7 @@ class TwoPointCV(CollectiveVariable):
     Parameters
     ----------
     indices : list[tuple(int)]
-       Must be a or groups (size 2) of atoms.
+       Must be a tuple (size 2) of atoms.
```

Comment:
```suggestion
       Must be a group (size 2) of atoms.
```

Reply:
These are indeed groups. In the line below in each case I tried to indicate that a group is specified as a list or a tuple. If you feel it's not clear enough we can certainly think of a way of improving it.

Reply:
Aaah, now I got it. OK I think it's just the extra "or" (`Must be a or groups (size 2)...`) that confused me. Let me commit your suggestions.